docs/concepts/index-patterns.asciidoc

```diff
@@ -5,22 +5,20 @@
 An index pattern selects the data to use and allows you to define properties of the fields.
 
 An index pattern can point to a specific index, for example, your log data from yesterday,
```

Comment:
This is accurate, but it may be simpler to say "An index pattern points to one or more indices, data streams, or aliases. For example, ..."